baccmod/base_fit_acceptance_map_creator.py

```diff
@@ -161,8 +161,9 @@ def fit_background(self, model: Model, *coords: List[np.ndarray], count_map: np.
 
         # Compute exposure correction
         mask = exp_map_total > 0
-        exp_correction = exp_map / exp_map_total
-        exp_correction[~mask] = 0
+        exp_map_total_safe = exp_map_total.copy()
+        exp_map_total_safe[mask] = 1
```

Comment:
It should be `exp_map_total_safe[~mask] = 1`, this probably completly breaks your model as is. I am preparing a PR now which will include the fix.